test/e2e/pages/vl-upload.page.js

```diff
@@ -1,12 +1,84 @@
 const VlUpload = require('../components/vl-upload');
 const { Page } = require('vl-ui-core').Test;
 const { Config } = require('vl-ui-core').Test;
+const { By } = require('selenium-webdriver');
```

Comment:
By zit ook in de vl-ui-core.Test.Setup. Denk dat we dat misschien best gebruiken indien mogelijk, aangezien vl-upload niet rechtstreeks een dependency zal hebben op selenium-webdriver.

Reply:
ok